tests/lib/rule-tester/rule-tester.js

```diff
@@ -731,6 +731,47 @@ describe("RuleTester", () => {
 		});
 	});
 
+	describe("assert valid test cases", () => {
+		/**
+		 * Runs valid tests
+		 * @param {Array<string|object>} valid Valid tests
+		 * @returns {void}
+		 */
+		function runValidTests(valid) {
+			ruleTester.run(
+				"no-eval",
+				require("../../fixtures/testers/rule-tester/no-eval"),
+				{ valid, invalid: [] },
+			);
+		}
+
+		it("Valid test case must not have 'errors' property", () => {
```

Comment:
Can you please the "valid" and "invalid" examples up in multiple test cases as this helps when debugging.
The same for the added tests for `output`.

Reply:
not sure I'm following.  Do you mean to add some comments?

Reply:
resolved in https://github.com/eslint/eslint/pull/20125/commits/3a9b615ac60f8e325862784b12c3c75d4cbb3e8f